gnomad_constraint/utils/constraint_basics.py

```diff
@@ -79,3 +86,79 @@ def prepare_ht_for_constraint_calculations(ht: hl.Table):
     # Filter out locus with undefined exome coverage
     ht = ht.filter(hl.is_defined(ht.exome_coverage))
     return ht
+
+
+def create_constraint_training_dataset(
+    exome_ht: hl.Table,
+    context_ht: hl.Table,
+    mutation_ht: hl.Table,
+    max_af: float = 0.001,
+    keep_annotations: Tuple[str] = (
+        "context",
+        "ref",
+        "alt",
+        "methylation_level",
+        "exome_coverage",
+    ),
+    pops: Optional[Tuple[str]] = (),
```

Comment:
I felt `pops` should be optional here, because `pops = None` makes more sense at `if pops:` than `()`. When the user doesn't want to analyze population and doesn't include the `--use-pop`, the pops can be `None` instead of `()`. If `Optional` is used, `pops` is `None`, otherwise it defaults to `()`.

Reply:
I think I modified it because I was getting an error with None during testing, if making it Optional works as you expect then it's fine to keep